.travis.yml

```diff
@@ -17,8 +17,19 @@ addons:
     packages:
     - libgconf-2-4
 script:
-  - npm run test
-  - npm run server &
+  # unit tests, also prints code coverage summary
+  - npm run test -- --coverageDirectory unit-coverage
+
+  # to enable code instrumentation for E2E tests set NODE_ENV=cypress
+  - NODE_ENV=cypress npm run server &
```

Comment:
What if we use your pkg (`start-server-and-test`) to run the server here? I've already had a problem with the server not starting before the cypress run script.

Reply:
I like the separate commands to start the server on IC because it separates the STDOUT output from the server from the STDOUT from Cypress. I find that simpler to read